mne/decoding/classifier.py

```diff
@@ -395,7 +395,7 @@ def transform(self, epochs_data, y=None):
 
         if self.l_freq is None and self.h_freq is not None:
             epochs_data = \
-                low_pass_filter(epochs_data, self.fs, self.h_freq,
+                low_pass_filter(epochs_data, self.info['sfreq'], self.h_freq,
```

Comment:
can you check why tests were passing with such an obvious bug and fix the tests?

thx